src/balancerd/src/codec.rs

```diff
@@ -212,39 +211,6 @@ impl Encoder<BackendMessage> for Codec {
     }
 }
 
-trait Pgbuf: BufMut {
-    fn put_string(&mut self, s: &str);
-    fn put_length_i16(&mut self, len: usize) -> Result<(), io::Error>;
-    fn put_format_i8(&mut self, format: mz_pgwire_common::Format);
-    fn put_format_i16(&mut self, format: mz_pgwire_common::Format);
-}
-
-impl<B: BufMut> Pgbuf for B {
-    fn put_string(&mut self, s: &str) {
-        self.put(s.as_bytes());
-        self.put_u8(b'\0');
-    }
-
-    fn put_length_i16(&mut self, len: usize) -> Result<(), io::Error> {
-        let len = i16::try_from(len)
-            .map_err(|_| io::Error::new(io::ErrorKind::Other, "length does not fit in an i16"))?;
-        self.put_i16(len);
-        Ok(())
-    }
-
-    fn put_format_i8(&mut self, format: mz_pgwire_common::Format) {
-        self.put_i8(match format {
-            mz_pgwire_common::Format::Text => 0,
-            mz_pgwire_common::Format::Binary => 1,
-        })
-    }
-
-    fn put_format_i16(&mut self, format: mz_pgwire_common::Format) {
-        self.put_i8(0);
-        self.put_format_i8(format);
-    }
-}
-
```

Comment:
@maddyblue this was duplicated in the `balancer`, `pgwire`, and `pgwire-common` crates. I removed it from `balancer` and `pgwire`, just want to make sure this was okay

Reply:
Yup! The hope was to remove the duplication at some point. I probably just forgot about it.